BREAKING.md

```diff
@@ -106,6 +81,28 @@ to be `string | undefined`.
 To improve bundle size, it will be converted to a free function so that this logic is tree-shakeable.
 The corresponding method on `IMergeTreeTexHelper` will also be removed.
 
+### Remove `type` field from `ShareLinkInfoType`
+This field has been deprecated and will be removed in a future breaking change. You should be able to get the kind of sharing link from `shareLinkInfo.createLink.link` property bag.
+
+### Remove `ShareLinkTypes` interface
+`ShareLinkTypes` interface has been deprecated and will be removed in a future breaking change. Singnature of `createOdspCreateContainerRequest` has been updated to now accept `ISharingLinkKind` property instead.
+```diff
+    function createOdspCreateContainerRequest(
+        siteUrl: string,
+        driveId: string,
+        filePath: string,
+        fileName: string,
+-       createShareLinkType?: ShareLinkTypes,
++       createShareLinkType?: ShareLinkTypes | ISharingLinkKind,
+    ):
+```
+
+
+### Remove `enableShareLinkWithCreate` from `HostStoragePolicy`
+`enableShareLinkWithCreate` feature gate has been deprecated and will be removed in a future breaking change. If you wish to enable creation of a sharing link along with the creation of Fluid file, you will need to provide `createShareLinkType:ISharingLinkKind` input to the `createOdspCreateContainerRequest` function and enable the feature using `enableSingleRequestForShareLinkWithCreate` in `HostStoragePolicy`
+### Add assertion that prevents sending op while processing another op
+`preventConcurrentOpSend` has been added and enabled by default. This will run an assertion that closes the container if attempting to send an op while processing another op. This is meant to prevent non-deterministic outcomes due to concurrent op processing.
+
 ## 2.0.0 Breaking changes
```

Comment:
@scottn12 could we rename titles as part of this PR to reflect *internal* ? I'm assuming all this should be 2.0.0-internal-1.0.0 as these are solely breaking changes ?

Reply:
Maybe we should do this in a separate PR, since we will need to sort through each item to properly place them into the correct section

Reply:
Here is that one: https://github.com/microsoft/FluidFramework/pull/12161. Given I ended up rearranging the announcements to correct sections, maybe we could take that one as base and merge it first ? Maybe it already encompasses the changes from here. Please give it a look, it would definitely be worthwhile another person giving it a deeper look